concepts/handlers.go

```diff
@@ -131,6 +132,57 @@ func (h *ConceptsHandler) GetConcept(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (h *ConceptsHandler) DeleteConcept(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	uuid := vars["uuid"]
+	conceptType := vars["concept_type"]
+
+	transID := transactionidutils.GetTransactionIDFromRequest(r)
+	w.Header().Add("Content-Type", "application/json")
+	w.Header().Set("X-Request-Id", transID)
+
+	// Validate that the concept exists and is of the right type.
+	obj, found, err := h.ConceptsService.Read(uuid, transID)
+	if err != nil {
+		writeJSONError(w, err.Error(), http.StatusServiceUnavailable)
+		return
+	}
+	if !found {
+		w.WriteHeader(http.StatusNotFound)
+		_, _ = w.Write([]byte(fmt.Sprintf("{\"message\":\"Concept with prefUUID %s not found in db.\"}", uuid)))
```

Comment:
`nitpick`: If we don't care about what w.Write() is returning why just not erase it?
```suggestion
		w.Write([]byte(fmt.Sprintf("{\"message\":\"Concept with prefUUID %s not found in db.\"}", uuid)))
```